app/models.py

```diff
@@ -2143,7 +2142,7 @@ class VAProfileLocalCache(db.Model):
     va_profile_id = db.Column(db.Integer, nullable=False)
     communication_item_id = db.Column(db.Integer, nullable=False)
     communication_channel_id = db.Column(db.Integer, nullable=False)
-    source_datetime = db.Column(db.DateTime, nullable=False)
+    source_datetime = db.Column(db.DateTime, nullable=True)
```

Comment:
Null safety question again.

Reply:
This field is never used in the codebase. I think it's safe to be nullable